cmd/fleetctl/query.go

```diff
@@ -131,11 +132,14 @@ func queryCommand() *cli.Command {
 				output = newJsonWriter(c.App.Writer)
 			}
 
-			hosts := strings.Split(flHosts, ",")
+			hostIdentifiers := strings.Split(flHosts, ",")
 			labels := strings.Split(flLabels, ",")
 
-			res, err := fleet.LiveQuery(flQuery, queryID, labels, hosts)
+			res, err := client.LiveQuery(flQuery, queryID, labels, hostIdentifiers)
 			if err != nil {
+				if strings.Contains(err.Error(), "no hosts targeted") {
```

Comment:
Leaving as reminder to add a test for this in case we change the error string in the server.